.pre-commit-config.yaml

```diff
@@ -9,27 +9,12 @@ repos:
     hooks:
       - id: black
         language_version: python3.7
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.2.1
+  - repo: local
     hooks:
     - id: prettier
-      args: [--write]
-      types_or: [javascript, jsx, ts, tsx]
+      name: prettier
+      language: system
+      types_or: [javascript,jsx,ts,tsx,markdown,html,scss,css,yaml]
       files: ^src/ts/src
-  - repo: https://github.com/pre-commit/mirrors-eslint
-    rev: v7.11.0
-    hooks:
-    - id: eslint
-      args: [--fix, --config src/ts/.eslintrc.js. --ignore-path src/ts/.eslintignore, --max-warnings 45]
```

Comment:
GH Action already checks lint for us, so we don't have to run `eslint` with every commit 🎉